packages/pyright-internal/src/analyzer/docStringConversion.ts

```diff
@@ -589,7 +589,7 @@ class DocStringConverter {
 
         // catch-all for styles except reST
         const hasArguments =
-            !line?.endsWith(':') && !line?.endsWith('::') && !!line.match(/^\s*.*?\w+(\s*\(.*?\))*\s*:\s*\w+/g);
+            !line?.endsWith(':') && !line?.endsWith('::') && !!line.match(/^\s*.*?\w+(\s*\(.*?\))*\s*:\s*\p{L}+/gu);
```

Comment:
probably we want to add some tests covering issue it is fixing to prevent regressions in future?

Reply:
that will be good, thank you.

Reply:
@Aruelius, are you planning to add tests to this PR?

Reply:
I actually have a fix.. i used this line
`!line?.endsWith(':') && !line?.endsWith('::') && !!line.match(/^\s*.*?\S+(\s*\(.*?\))*\s*:\s*\S+/g);`

Reply:
@bschnurr, do you mean that you're going to update this PR? Or that you're going to fix the problem in a separate PR and we should close this one?

Reply:
`\S` includes non-letter characters like `<`, `>`, `+`, `$`, `,`, `'`, etc, whereas `\p{L}` only includes characters that Unicode says are letters. Is `\S` what we want? I'm not familiar with the docstring format requirements, but going from `\w` to `\S` seems strange to me unless we should always have been including symbol characters.

Btw, my initial suggestion of `\p{L}` only includes letters, not numbers. So if we wanted to use that approach, I believe `[\p{L}\p{N}]` would be better.

Reply:
I thought using `\S` would be the most flexible when it comes to user naming stuff